src/Init/Control/Basic.lean

```diff
@@ -131,7 +143,7 @@ Once it's been through `foo` we can then unpack the state again with `restoreM`.
 Hence we can apply `foo` to `bar` without losing track of the state.
 
 Here `stM β = σ × β` is the 'packaged result state', but we can generalise:
-if we have a tower `StateT σ₁ <| StateT σ₂ <| IO`, then we can get the
+if we have a tower `StateT σ₁ <| StateT σ₂ <| IO`, then the
 composite packaged state is going to be `stM₁₂ β := σ₁ × σ₂ × β` or `stM₁₂ := stM₁ ∘ stM₂`.
 
 Now we can define `MonadControl m n`. Call `m` the 'base monad', in the above example it was `IO`.
```

Comment:
How about something like the following instead the sentence below, to differentiate it from `MonadFunctor`:
> `MonadControl m n` means that when programming in the monad `n`, we can switch to the base monad `m` using `control`, just like with `liftM`. In contrast to `liftM`, however, we also get a function `runInBase` that allows us to "lower" actions in `n` into `m`.

This is still quite simplified, it doesn't address `stM` or `restoreM`.

Reply:
Yeah there is a link to a more detailed tutorial so it doesn't need to be super detailed.